cmd/vsphere-csi/main.go

```diff
@@ -38,9 +48,19 @@ func main() {
 		fmt.Printf("%s\n", service.Version)
 		return
 	}
-	log := logger.GetLoggerWithNoContext()
+	logType := logger.LogLevel(os.Getenv(logger.EnvLoggerLevel))
+	logger.SetLoggerLevel(logType)
+	ctx, log := logger.GetNewContextWithLogger()
 	log.Infof("Version : %s", service.Version)
 
+	// Set CO Init params
+	clusterFlavor, err := csiconfig.GetClusterFlavor(ctx)
+	if err != nil {
+		log.Errorf("Failed retrieving cluster flavor. Error: %v", err)
+	}
+	common.SetInitParams(ctx, clusterFlavor, &service.COInitParams, *supervisorFSSName, *supervisorFSSNamespace,
```

Comment:
Same as above. If `clusterFlavor` == ''" , `SetInitParams` is not handling this case.

Reply:
Handled it as above by adding a `Fatal` error message in `SetInitParams`